### ISSUE TITLE
[BUG] Histograms no visible in the dashboard

### ISSUE THREAD

**Issue by nlgranger** [gradio-app/trackio#637]
The dashboard does not show histogram metrics (trackio==0.31.5).

# Expected behavior

After using `trackio.log({"logits": trackio.Histogram(x)})`, a histogram chart should appear in the dashboard.

# Actual behavior

The histogram is stored in the sqlite database.

```json
{
    "logits": {
        "_type": "trackio.histogram",
        "bins": [ 7.988692283630371, "...", 12.998191833496094 ],
        "values": [ 1, "...", 1 ]
    }
}
```

But no histogram appears in the web ui. There is nothing in the javascript console or in the terminal indicating an error.
